build-logic/testing-convention/src/main/kotlin/convention.gradle-testing.gradle.kts

```diff
@@ -34,12 +34,40 @@ val gradleTestTask = tasks.register<Test>("gradleTest") {
 
     useJUnitPlatform()
 
-    @Suppress("MagicNumber")
-    maxParallelForks = 8
+    /**
+     * timings provided only as relative to each other
```

Comment:
> relative to each other

Didn't get what it means. 
Is it about optimizing only one parameter?

Reply:
How do you measure?

Reply:
i have commited a performance scenario for gradle profiler and also redone the measurements with more accurate results